src/pages/UsedMarket/components/PaginationBar.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import "./PaginationBar.css";
+import { ReactComponent as LeftArrow } from "../../../assets/images/icons/arrow_left.svg";
+import { ReactComponent as RightArrow } from "../../../assets/images/icons/arrow_right.svg";
+
+const PaginationBar = ({ totalPageNum, activePageNum, onPageChange }) => {
+    const maxVisiblePages = 5;
```

Comment:
상수값은 컴포넌트 밖에 선언해주세요!
```suggestion
    const MAX_VISIBLE_PAGE = 5;
```